frontend/src/app/restaurant/page.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+interface Restaurant {
```

Comment:
@seoulection already created a type `Restaurant` in the [PR](https://github.com/8l-project-jam/tastemate/blob/f9ff35d3857955d529f2a2c917ba90889c96b145/frontend/src/types/index.ts). Let's use that type and add these properties.

Reply:
@ksdenisova thanks for pointing this out!